sys/kern/device.c

```diff
@@ -91,3 +91,14 @@ resource_t *device_take_resource(device_t *dev, res_type_t type, int rid,
 
   return rle->res;
 }
+
+device_t *device_if_find_impl(device_t *dev, size_t iface,
+                              size_t method_offset) {
+  while ((!dev->parent->driver->interfaces[iface]) ||
+         (!*(vaddr_t **)(dev->parent->driver->interfaces[iface]
```

Comment:
I think that `void **` is more pertinent.

Reply:
Yes, probably. I'll change it to `void **`.

Reply:
Done!